.github/workflows/cluster-new.yml

```diff
@@ -49,4 +51,12 @@ jobs:
         shell: bash
         # we do not compile client-cpp for saving time, it is tested in client.yml
         # we can skip influxdb-protocol because it has been tested separately in influxdb-protocol.yml
-        run: mvn -B clean verify -Dsession.test.skip=true -Diotdb.test.skip=true -Dcluster.test.skip=true -Dtsfile.test.skip=true -pl integration -am -PCluster
+        run: mvn -B clean verify -Dsession.test.skip=true -Diotdb.test.skip=true -Dcluster.test.skip=true -Dtsfile.test.skip=true -Dcommons.test.skip=true -Dconfignode.test.skip=true -pl integration-new -am -PNewCluster
+      - name: Upload Artifact
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cluster-log
+          path: integration-new/target/cluster-logs
+          retention-days: 1
```

Comment:
This will upload the test log files?

Reply:
Yes. This upload step only happens after the previous IT step fails.
You can download the zip file of log files at [this URL](https://github.com/irvine0109/iotdb/actions/runs/2333221271). The Artifacts part is at the bottom.